openpgp/uat.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package openpgp
+
+import (
+	"crypto"
+	"io"
+
+	"github.com/ProtonMail/go-crypto/openpgp/errors"
+	"github.com/ProtonMail/go-crypto/openpgp/packet"
+)
+
+func (t *Entity) AddPhoto(jpegBytes []byte, config *packet.Config) error {
+	creationTime := config.Now()
+	keyLifetimeSecs := config.KeyLifetime()
+
+	uat, err := packet.NewUserAttributePhotoBytes([][]byte{jpegBytes})
```

Comment:
Shouldn't we allow passing multiple photos?

Reply:
i think that if they want to pass multiple photos, user need call this func multiply times

Reply:
But then you get multiple User Attribute packets with one photo each, rather than one User Attribute packet with multiple photos. I'm not really sure if that matters, but since the underlying functions support it, perhaps we might as well expose it?

Reply:
ok